test/new-e2e/.vscode/tasks.json.template

```diff
@@ -5,7 +5,7 @@
         {
             "label": "aws-vault-wrap",
             "type": "shell",
-            "command": "aws-vault export sso-agent-sandbox-account-admin > .vscode/aws-vault-wrap.env",
+            "command": "aws-vault export sso-agent-sandbox-account-admin > ${workspaceFolder}/.vscode/aws-vault-wrap.env",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Does this wrap terminal only, or also go.tools ?

Reply:
The config is not new, it was just not fully working. It's only used for `debug` run.